index.html

```diff
@@ -54,143 +54,14 @@ <h1 class="head1"> I’m George<br>
-                  <div class="grid-item" id="grey-works"> <img src="img/Counter.png" alt="counter" class="counter"> 2015</div>
-               </div>
-               <div class="doc">
-                  <p>
-                    Experimental content creation feature that allows users to add
-                     to an existing story over the course of a day without spamming their friends.                    
-                  </p>
-               </div><br>
-               <div class="tags">
-                  <ul class="works">
-                     <li> <a href="#" class="btn"> html </a></li>
-                     <li> <a href="#" class="btn"> css </a></li>
-                     <li> <a href="#" class="btn"> javascript </a></li>
-                     <li> <a href="#" class="btn" id="rub"> Ruby On Rails</a></li>
-                  </ul>
-               </div>
-               <a href="#">
-                  <div class="btn-main">See Project</div>
-               </a>
-            </section>
-         </article >
-         <!--Snap 4  -->
-         <article  class="snapshot1">
-            <img src="img/Snapshoot Portfolio4.png" alt="snapshot1" class="image-po">
-            <section class="main" id="snap-item">
-               <div class="head-one-alt">
-                  <h2> Uber Navigation
-                  </h2>
-               </div>
-               <div class="grid-container">
-                  <div class="grid-item"> <span class="canop">Uber</span></div>
-                  <div class="grid-item" id="grey-works"> <img src="img/Counter.png" alt="counter" class="counter"> Lead Developer</div>
-                  <div class="grid-item" id="grey-works"> <img src="img/Counter.png" alt="counter" class="counter"> 2018</div>
-               </div>
-               <div class="doc">
-                  <p>
-                    A smart assistant to make driving more safe, efficient,
-                     and fun by unlocking your most expensive computer: your car.                    
-                  </p>
-               </div><br>
-               <div class="tags">
-                  <ul class="works">
-                     <li> <a href="#" class="btn"> html </a></li>
-                     <li> <a href="#" class="btn"> css </a></li>
-                     <li> <a href="#" class="btn"> javascript </a></li>
-                     <li> <a href="#" class="btn" id="rub"> Ruby On Rails</a></li>
-                  </ul>
-               </div>
-               <a href="#">
-                  <div class="btn-main">See Project</div>
-               </a>
-            </section>
-            <img src="img/Snapshoot-Portfolio-DESK3.png" alt="snapshot1" class="image-desk" id="snap-item" >
-         </article >
+      
+<!-- popup begins -->
+      <section class="popup-window"></section>
+      <section class="works_section" id="works"></section>
```

Comment:
- [ ] It would be great if the grab the `work_section` in the popup section. [ OPTIONAL]
- [ ] Kindly consider adding a scroll bar when the card popups so that the user may be able to see all the contents in the pop-up model.
`hint: You may need to grab your pop-up in a container and set the ::-webkit-scrollbar to it.`
As you can see on your design :point_down: 
The user is not able to scroll so that she/he may see all the content in the pop-up modal
![image](https://user-images.githubusercontent.com/97100091/196737336-9088c1d5-d060-4426-ab1b-96901d58285e.png)

Reply:
You can check your images folder. you have `Counter.png` but in your code your write  `counter.png` that is why your image still broken
I am talking about  those little images on the header of your popup model

Reply:
I understand now, thank you